keyboards/planck/keymaps/solstice22/readme.md

```diff
@@ -0,0 +1,2 @@
+# The Default Planck Layout
```

Comment:
You may want to update or remove this, since .... well, it's not the default layout.